extra_data/sourcedata.py

```diff
@@ -203,9 +206,73 @@ def _only_tids(self, tids) -> 'SourceData':
                 if f.has_train_ids(tids, self.inc_suspect_trains)
             ] or [self.files[0]],
             section=self.section,
+            is_single_run=self.is_single_run,
             inc_suspect_trains=self.inc_suspect_trains
         )
 
+    def run_metadata(self) -> Dict:
+        """Get a dictionary of metadata about the run
+
+        From file format version 1.0, the files capture: creationDate,
+        daqLibrary, dataFormatVersion, karaboFramework, proposalNumber,
+        runNumber, sequenceNumber, updateDate.
+        """
+        if not self.is_single_run:
+            raise MultiRunError()
+
+        return self.files[0].metadata()
+
+    def run_value(self, key):
+        """Get a single value from the RUN section of data files.
+
+        This method is intended for use with data from a single run. If you
+        combine data from multiple runs, it will raise MultiRunError.
+
+        Returns the RUN parameter value corresponding to the *key* argument.
+        """
+        print('sourcedtata', self.is_single_run)
+        if not self.is_single_run:
+            raise MultiRunError()
+
+        if not self._is_control:
+            raise SourceNameError(self.source)
```

Comment:
This roughly made sense for `run.get_run_value(src, key)`, but I think it would be surprising for `src.run_value(key)` to raise SourceNameError. Maybe we should have a generic exception like ValueError with an explanation that instrument sources don't have RUN data.